### DISCUSSION TITLE
When will the next version of django-rest-framework-json-api with support for Django 4.2 be released on PyPI?

### DISCUSSION THREAD

**Discussion by elanqo** [django-json-api/django-rest-framework-json-api#1170] [Q&A] [answered]
I see that the changes to support Django 4.2 have already been merged into the djangorestframework-jsonapi repository on GitHub, but I'm not sure when the next release will be published to PyPI.

Does anyone know when the next release of djangorestframework-jsonapi 4.2 is expected to be released? I'm eager to start using it in my project!

**Comment by sliverc** [comment, marked as answer]
I was hoping that DRF will have 3.15 released by now, but it seems it is taking a bit longer than anticipated.

See discussion here #1142. It seems it should be ready [soon though](https://github.com/encode/django-rest-framework/pull/8794#issuecomment-1667278047). I will wait a few more days if it won't be released we will do a release of DJA and then do one right after DRF 3.15 is released as well.

**Reply by elanqo** [reply]
@sliverc thanks for the reply.

**Reply by sliverc** [reply]
It is taking longer for DRF 3.15 to be released as anticipated. So DJA 6.1.0 is released [now](https://github.com/django-json-api/django-rest-framework-json-api/releases/tag/v6.1.0).

**Reply by elanqo** [reply]
Thanks for the info.

**Reply by arttuperala** [reply]
Is the next release (v6.2.0?) also pending for DRF 3.15? Because DRF 3.15 seems to be still stalled and there's some unreleased bug fixes that might be good to have in the PyPI version.
